### PR TITLE
[ticket/5] [WIP] phpBB-Blog build script

### PR DESCRIPTION
**Pull request by erikfrerejean**
The phpBB-Blog build script will be based around the Support Toolkit
build module, this commit simply adds the complete build package
from the STK.

TICKET-#5

### CODE REVIEW

File: build/builder.php [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+ */
+if (php_sapi_name() != 'cli')
+{
+	die("This program must be run from the command line.\n");
+}
+
+/**
+ * The main STK Build class
+ */
+class stk_builder
```
Comment (imkingdavid) [inline build/builder.php:21]: Yes, it's based on the STK builder, but shouldn't we rename it for our purposes?

**Comment by michaelcullum** [comment]
This is WIP, all he's done is upload the STK one, he hasn't done anything too it yet.

**Comment by michaelcullum** [comment]
Bump

**Comment by erikfrerejean** [comment]
Is the spec finalised?

**Comment by michaelcullum** [comment]
spec for?

**Comment by erikfrerejean** [comment]
How the packages must be "build". Dir structure and so, by my understanding that wasn't finalised when I left. So no use to do anything here until thats finalised

**Comment by michaelcullum** [comment]
Ah ok. I've spoken with the MOD team and they agree that packages will be the files inside /ext/ however, recent changes to the general extensions overall plan is extensions will be located in ext/vendor/package/ So we need to get the package and put it in /phpbb-blog/phpbb-blog/fileshere.php and then that would be uploaded into /ext/ by the forum owner (upload form on extensions admin will most likely be added, depends how much work it would need and whether I have time before 3.1-Alpha)

**Comment by erikfrerejean** [comment]
Is there any way to get rid off the second `phpbb-blog`? I've seen with the STK that having a nested directory with the same name confuses people and they will upload the top one rather than the nested one. For example include the version number in the top one. ex: `phpbb-blog_1.0.0/phpbb-blog/somefile.php`

**Comment by michaelcullum** [comment]
Nope, its vendor/packagename

**Comment by imkingdavid** [comment]
Then let's rename the package name to just blog. So it would be phpbb-blog/blog.

**Comment by michaelcullum** [comment]
Ok, sounds good to me.

**Comment by erikfrerejean** [comment]
+1

**Comment by michaelcullum** [comment]
I'll close this for now as its basically on the backburner. @erikfrerejean when you want to finish it just re-open this pr.

**Pull request closed without merging**